packages/app-project/src/screens/ClassifyPage/components/FinishedForTheDay/FinishedForTheDayContainer.js

```diff
@@ -19,11 +20,16 @@ class FinishedForTheDayContainer extends Component {
   render () {
     const { imageSrc, isLoggedIn, projectName } = this.props
     return (
-      <FinishedForTheDay
-        imageSrc={imageSrc}
-        isLoggedIn={isLoggedIn}
-        projectName={projectName}
-      />
+      <ResponsiveContext.Consumer>
```

Comment:
Is it worth creating a HOC for ResponsiveContext, so we can export the unwrapped component and preserve the tests?

Reply:
Sure, but I would prefer as a separate update.

Reply:
Actually I ended up adding a HOC to the react-components library much like the one for ThemeContext because I might use it again to fix the last padding issue.